.copr/prepare.sh

```diff
@@ -34,6 +34,6 @@ if [ ! -d conmon ]; then
     git clone -n --quiet https://github.com/containers/conmon
 fi
 pushd conmon
-git checkout --detach 6f3572558b97bc60dd8f8c7f0807748e6ce2c440
+git checkout --detach d532caebc788fafdd2a305b68cd1983b4039bea4
```

Comment:
without a more recent commit hash, I got:
```
#Error: could not get runtime: please update to v2.0.1 or later: outdated conmon version
```

so i took the most recent one on master for conmon. Please advise if it is wrong.

Reply:
SGTM